e2e/app/run.go

```diff
@@ -174,15 +174,16 @@ func E2ETest(ctx context.Context, def Definition, cfg E2ETestConfig) error {
 		return err
 	}
 
+	stopReceiptMonitor := StartMonitoringReceipts(ctx, def)
+
 	var eg errgroup.Group
 	eg.Go(func() error { return testGasPumps(ctx, def) })
 	eg.Go(func() error { return testBridge(ctx, def) })
+	eg.Go(func() error { return xbridge.Test(ctx, NetworkFromDef(def), ExternalEndpoints(def)) })
```

Comment:
Put here because it was helpful in debugging. Also, putting in `e2e/tests/` caused lot of nonce conflicts with solver tests. Something to think about. Solution can be part of the on going discussion on how we should structure e2e tests going forward